src/admin-views/seating/components/layout-card.php

```diff
@@ -4,5 +4,38 @@
  *
  * @since TBD
  *
- * @var Layout_Card $card The Seat Layout card to render.
+ * @var Object $card The Seat Layout card to render.
  */
+
+?>
+
+<div class="tec-tickets__tab__cards__item">
+	<div class="tec-tickets__tab__cards__item-thumbnail">
+		<img src="#" alt="<?php echo esc_attr( $card->name ); ?>">
+	</div>
+	<div class="tec-tickets__tab__cards__item-content">
+		<div class="tec-tickets__tab__cards__item-title"><?php echo esc_html( $card->name ); ?></div>
+		<div class="tec-tickets__tab__cards__item-capacity">
+			<?php
+			echo esc_html(
+				sprintf(
+					/* translators: %s: the capacity of the layout */
+					_x( '%s seats', 'map seats count', 'event-tickets' ),
+					number_format_i18n( $card->seats )
+				)
+			);
+			?>
+		</div>
+		<div class="tec-tickets__tab__cards__item-actions">
+			<a class="button button-secondary add-map" href="#">
+				<?php esc_html_e( 'Create Layout', 'event-tickets' ); ?>
```

Comment:
This is a Layout card component, it makes sense  to talk about layout, but will it make sense in the context of maps?